internal/pkg/cli/env_init.go

```diff
@@ -461,22 +461,28 @@ https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-er
 		if err != nil {
 			if err == selector.ErrSubnetsNotFound {
 				log.Errorf(`No existing public subnets were found in VPC %s. You can either:
-- Create new public subnets and then import them.
-- Use the default Copilot environment configuration.`, o.importVPC.ID)
+- %s to interrupt, create new public subnets, then rerun %s and import them.
+- %s to interrupt, then rerun %s and use the default Copilot environment configuration.
+- Proceed without public subnets, knowing that deploying a load-balanced web service in this environment will fail because Load Balancers require at least two public subnets in different Availability Zones.
+`, o.importVPC.ID, color.HighlightUserInput("ctrl-c"), color.HighlightCode("copilot env init"), color.HighlightUserInput("ctrl-c"), color.HighlightCode("copilot env init"))
+			} else {
+				return fmt.Errorf("select public subnets: %w", err)
 			}
-			return fmt.Errorf("select public subnets: %w", err)
 		}
 		o.importVPC.PublicSubnetIDs = publicSubnets
```

Comment:
can we force them to select at least 2 public subnets if they choose at least one? and avoid the ALB failure altogether.
```
if len(publicSubnets) > 0 && len(publicSubnets) <2
  throww err "select public subnets: at least 2 public subnets must be selected to support Load Balanced Web Services"
```

Reply:
What do you think of adding this msg to hint that they can skip importing public subnets even if they have some:
```
Which public subnets would you like to use?
You can press enter to skip choosing public subnets if you don't want to deploy any internet-facing web services.
> [ ] subnet-f468b08c
> [ ] subnet-0e1b3925
```